helm/thingsboard/values.yaml

```diff
@@ -144,6 +152,11 @@ mqtt:
     #pullPolicy: Always
     #tag: "X.Y.Z"
   replicaCount: 1
+  log:
+    root:
+      level: WARN
+    loggers:
+      - <logger name="org.thingsboard.server" level="WARN" />
```

Comment:
At this level it does not look right. Info messages usually are the informative ones, that gives useful information during service run. Please set it to warn during performance testing only if something is too verbose.

Reply:
Ok, I'll do another PR

Reply:
https://github.com/midokura/thingsboard-ce-k8s/pull/74